### PR TITLE
New page for analyzer message IST0002

### PR DESCRIPTION
**Pull request by adammil2000**

### CODE REVIEW

File: content/en/blog/2019/announcing-istio-client-go/index.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ for developers to create controllers and perform Create, Read, Update and Delete
 
 This was a highly requested functionality by many Istio users, as is evident
 from the feature requests on the clients generated by [Aspen Mesh](https://github.com/aspenmesh/istio-client-go)
-and the [Knative project](https://github.com/knative/pkg/tree/master/apis/istio).
+and the [Knative project](https://github.com/knative/pkg).
```
Comment (clarketm) [inline content/en/blog/2019/announcing-istio-client-go/index.md]:
```suggestion
and the [Knative project](https://github.com/knative/pkg/tree/7325e901c0768a6d3b1b244ecd021a88d45644a4/apis/istio).
```

Wouldn't a permalink to the Knative Istio client at a commit *before* it was removed  make for sense then linking to an arbitrary `pkg` directory with the code removed?

File: content/en/docs/reference/config/analysis/ist0002/index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+title: Deprecated
+layout: analysis-message
+---
+
+This message occurs when you use a function that is planned for removal in a future version.
```
Comment (geeknoid) [inline content/en/docs/reference/config/analysis/ist0002/index.md]: "function" is an overloaded term. How about "feature" instead?

File: content/en/blog/2019/announcing-istio-client-go/index.md (hunk 1 above)
Comment (geeknoid) [reply content/en/blog/2019/announcing-istio-client-go/index.md]: Let's remove this fix, it's been addressed already.

File: content/en/docs/reference/config/analysis/ist0002/index.md (hunk 2 above)
Comment (adammil2000) [reply content/en/docs/reference/config/analysis/ist0002/index.md]: Great, done!

**Pull request merged**